apps/gateway/lib/gateway/router.ex

```diff
@@ -9,6 +9,12 @@ defmodule Gateway.Router do
     pipe_through :api
   end
 
+  scope "/auth", Gateway do
+    pipe_through :api
+
+    get "/:provider/token/:token_id", Controllers.AuthController, :validate_token
```

Comment:
Either pattern match `:provider` in `validate_token/2` or remove it

Reply:
Done!